classes/test/testcase_helper_trait.php

```diff
@@ -148,4 +158,33 @@ protected function require_mock_server(): void {
             );
         }
     }
+    // phpcs:disable moodle.NamingConventions.ValidFunctionName.LowercaseFunction
```

Comment:
Why? These are already part of the advanced_testcase

Reply:
I retract my statement :)
It seems that in PHP storm, if the trait is used somewhere in a class that inherit from the right methods, it will not show an error. That's strange, but if this is true, that's quite clever...